isv.md

```diff
@@ -11,6 +11,7 @@ Additionally:
 
 | Category | ISV | Product | Resources |
 | --- | --- | --- | --- |
+| Analytics | [Anaconda](https://partners.amazonaws.com/partners/0010L00001qFzbyQAC) | [Anaconda](https://www.anaconda.com/products/commercial-edition) | Release: https://docs.anaconda.com/anaconda/install/graviton2/, https://repo.anaconda.com/pkgs/main/linux-aarch64/<br>Announcement: https://www.anaconda.com/blog/anaconda-aws-graviton2 |
```

Comment:
Typically people think of Anaconda as Data Science and ML tool,  less about analytics which typically mean data analytics .     i suggest we create a new category for data science and ML

Reply:
Updated the category to Data Science & ML. Thanks for the great suggestion!